fdm-app/app/routes/farm.$b_id_farm.$calendar.nutrient_advice.$b_id.tsx

```
@@ -211,8 +227,42 @@ function FieldNutrientAdvice({
     traceNutrients: NutrientDescription[]
 }) {
     const { field, calendar, nutrientsDescription } = loaderData
-    const asyncData = use(loaderData.asyncData)
+    const serverAsyncData = use(loaderData.asyncData)
     const location = useLocation()
+    const [searchParams, setSearchParams] = useSearchParams()
+
+    const fieldNutrientAdviceCache = useFieldNutrientAdviceCache()
+    const cachedData = fieldNutrientAdviceCache.get(field.b_id)
+    useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on asyncData.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)